app/pages/ServiceDiscoveryResults/SearchResults/SearchResults.scss

```diff
@@ -26,22 +38,46 @@
   .title {
     font-weight: bold;
     font-size: 20px;
+
+    @media screen and (max-width: 600px) {
+      font-size: 18px;
+      width: 320px;
+    }
   }
-  
+
   .address {
     font-size: 18px;
     margin-top: 12px;
+
+    @media screen and (max-width: 600px) {
+      width: 320px;
+      font-size: 14px;
```

Comment:
I think generally, the smallest font size that people recommend on mobile views is 16px. Any smaller and I think this will exclude older users or visually impaired users who may not have as keen eyesight.